### PR TITLE
Workings

### PR DESCRIPTION
**Pull request by Urchmaney**

### CODE REVIEW

File: src/logics/calculate.js [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
-    case '*':
-    case '/':
-    case '%':
-      obj.total = operate(calObj.total, calObj.next, btnName);
-      break;
-    default:
+  if (ops.includes(obj.operation)) {
+    obj.total = operate(obj.total, obj.next, obj.operation);
+  } else {
+    obj.total *= -1;
```
Comment (shloch) [inline src/logics/calculate.js:10]: Think about the tweak you'll apply to fix a very little bug on the app's logic

When i open your calculator app, if i click on a number (say `"5"`), then i click on `"="`, i will get ` -5 ` at the display

**Review by shloch — CHANGES_REQUESTED** [review]
### Hello @Urchmaney , 
i like your coding style, the manner in which you put functions that will help the components just above it..... i learnt it myself from looking at your code.....awesome !!

You can also delete the following unused files 
- `src/app.js`

my main focus is the little bug above

Please fix it and file for a re-view and we'll be done 🎅🎅🎅

**Comment by Urchmaney** [comment]
Thanks I appreciate.

**Comment by Urchmaney** [comment]
Submitted for another code review.
Good Work.

File: src/logics/calculate.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const calculate = (calObj, btnName) => {
   const obj = calObj;
   if (ops.includes(obj.operation)) {
     obj.total = operate(obj.total, obj.next, obj.operation);
-  } else {
+  } else if (obj.operation === '+/-') {
     obj.total *= -1;
```
Comment (shloch) [inline src/logics/calculate.js]: cool

**Review by shloch — APPROVED** [review]
### Nice job bro  👏👏 

You can merge this PR ....

**Pull request merged**